source/concurrent/Promise.ooc

```diff
@@ -33,6 +33,13 @@ Promise: abstract class {
 	start: static func (action: Func) -> This {
 		_ThreadPromise new(action)
 	}
+	createEmpty: static func -> This { _EmptyPromise new() }
```

Comment:
I don't know, I'm sure it is at least for one class.
@marcusnaslund what does our coding rules have to say about this ?